src/components/Dashboard.js

```diff
@@ -1,99 +1,15 @@
-import { gql, useQuery, useMutation } from '@apollo/client';
+import { gql, useQuery } from '@apollo/client';
 import { useAuth } from '../contexts/AuthContext';
-
+import { GET_WELLNESS_SCORE } from "../graphql/queries"
 const Dashboard = () => {
-    const [wellnessScore,setWellnessScore]=useState(0)
-    const [autonomyScore,setAutonomyScore]=useState(0)
-    const [developmentScore,setDevelopmentScore]=useState(0)
-    const [impactScore,setImpactScore]=useState(0)
-    const [leaderSupportScore,setLeaderSupportScore]=useState(0)
-    const [peerRelationsScore,setPeerRelationsScore]=useState(0)
-    const [workLoadScore,setWorkLoadScore]=useState(0)
-    const [lineScore, setLineScore]=useState(0)
     const { currentUser } = useAuth()
     const email=currentUser.email
-
-    const queryStringLine = gql`
-    query ($email:String!){
-        Tests(where: {
-            User:{
-              email:{
-                _eq:$email
-              }
-            }
-          }){
-            score
-          }
-        }`
-    
-    const queryStringWellness = gql`
-    query ($email:String!){
-        Tests(where: {
-              User:{
-                email:{
-                  _eq:$email
-                }
-              }
-            }){
-              
-              autonomy_score
-              development_score
-              impact_score
-              leader_support_score
-              peer_relations_score
-              work_load_score
-              score
-            
-            }
-          }`
-    
-
-    const Score = async () => {
-        try{
-            const { loading, error, data } = await useQuery(queryStringWellness,{variables:{email:email}})
-            if (loading) return null;
-            if (error) return `Error! ${error}`;
-            console.log(autonomyScore)
-            console.log(wellnessScore)
-            setAutonomyScore(data.Tests[0].autonomy_score)
-            //setDevelopmentScore(data.Tests[0].development_score)
-            // setImpactScore(data.Tests[0].impact_score)
-            // setLeaderSupportScore(data.Tests[0].leader_support_score)
-            // setPeerRelationsScore(data.Tests[0].peer_relations_score)
-            // setWorkLoadScore(data.Tests[0].work_load_score)
-            setWellnessScore(data.Tests[0].score)
-            // //function ListScore(props){
-                //const Tests=data.Tests;
-                //const scoreList = Tests.map((scOre.score));
-                //console.log(scoreList)
-            //}
-
-        }
-        catch(e){
-            console.log("this")
-            setWellnessScore(0)
-            setAutonomyScore(0)
-            setDevelopmentScore(0)
-            setImpactScore(0)
-            setLeaderSupportScore(0)
-            setPeerRelationsScore(0)
-            setWorkLoadScore(0)
-            setLineScore(0)
-        }
-
-	}
-    Score()
-
+    const {loading ,error,data} = useQuery(GET_WELLNESS_SCORE,{
+        variables:{email}
+    })
```

Comment:
This needs to be called in a function outside of the component. 
You are doing the api call within the component, even though your component depends on data from the api call.

Warning: Failed prop type: The prop `data` is marked as required in `ChartComponent`, but its value is `undefined`.